src/sql/src/parser.rs

```diff
@@ -147,13 +147,17 @@ impl<'a> ParserContext<'a> {
                     actual: self.peek_token_as_string(),
                 })?;
         ensure!(
-            !table_name.0.is_empty(),
+            !table_idents.0.is_empty(),
             InvalidTableNameSnafu {
-                name: table_name.to_string(),
+                name: table_idents.to_string(),
             }
         );
+        let (catalog_name, schema_name, table_name) = table_idents_to_full_name(&table_idents)?;
+
         Ok(Statement::ShowCreateTable(ShowCreateTable {
-            table_name: table_name.to_string(),
+            schema_name,
+            catalog_name,
+            table_name,
```

Comment:
```suggestion
            catalog_name,
            schema_name,
            table_name,
```

Reply:
Need it unit test?

Reply:
Or any other approach we can ensure the order of the arguments.

Reply:
Good eyes! 

Perhaps the ordering has no influence here. They are treated as struct fields rather than string formatting etc., and rust can handle this situation unless we write `catalog_name: schema_name`.

Reply:
understood, thnx